paket.lock

```diff
@@ -1,10 +1,15 @@
 NUGET
   remote: http://www.nuget.org/api/v2
   specs:
+    Dynamitey (1.0.2)
     Expression.Blend.Sdk (1.0.2)
     Fantomas (2.1)
       FSharp.Compiler.Service (>= 2.0.0.8)
     FSharp.Compiler.Service (2.0.0.8)
+    FSharp.Core (4.0.0.1)
```

Comment:
Yes, it's very worrying. However, I think it's time to switch to F# 4 and FSharp.Core anyway as it has some optimizations which are always very welcome in this project. It's a quite unrelated suggestion though.